meinberlin/assets/scss/style_user_facing.scss

```diff
@@ -60,47 +31,40 @@
 @import "components/embed_layout";
 @import "components/embed_status";
 @import "components/export_dashboard";
-@import "components/filter_bar";
-@import "components/follow";
 @import "components/hero_unit";
-@import "components/input_group";
-@import "components/item_detail";
 @import "components/item_detail_2";
 @import "components/label";
-@import "components/list_item";
-@import "components/live_questions";
-@import "components/lr_bar";
 @import "components/map";
 @import "components/maplist";
-@import "components/moderator_status";
 @import "components/module";
 @import "components/multiform";
-@import "components/pagination";
 @import "components/timeline";
 @import "components/poll";
-@import "components/pill";
-@import "components/idea_remark";
 @import "components/participation_tile";
 @import "components/progress";
 @import "components/project_header";
 @import "components/radio";
 @import "components/rating";
 @import "components/select2";
 @import "components/select-item";
-@import "components/select_dropdown";
 @import "components/simple_page";
 @import "components/status_bar";
-@import "components/storefront";
 @import "components/switch";
 @import "components/table";
 @import "components/tabs";
 @import "components/tile";
 @import "components/timeline-carousel";
-@import "components/typeahead";
-@import "components/upload";
-
-@import "print";
-@import "utility";
-@import "shame";
 
 @import "../extra_css/swiper";
+
+
+// New styles begin
+@import "../berlin_css/berlin_marketing";
+
+@import "styles_user_facing/utility";
+
+@import "components_user_facing/card";
+@import "components_user_facing/copyright";
+@import "components_user_facing/follow";
+@import "components_user_facing/moderator_status";
+@import "components_user_facing/pill";
```

Comment:
These are the new clean styles that have nothing to do with dashboard and are using BO based class names